certified-connectors/personr/personr.swagger.json

```diff
@@ -0,0 +1,491 @@
+                    }
+                  },
+                  "description": "response"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api-applicant-details": {
+      "post": {
+        "summary": "Retrieve Applicant Details",
+        "description": "Use this endpoint to get an applicant's details.",
+        "operationId": "Api-applicant-details",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "applicantId": {
+                  "type": "string",
+                  "description": "applicantId",
+                  "title": "Applicant ID"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "response": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "phone": {
+                      "type": "string",
+                      "description": "phone"
+                    },
+                    "nameFirst": {
+                      "type": "string",
+                      "description": "nameFirst"
+                    },
+                    "nameLast": {
+                      "type": "string",
+                      "description": "nameLast"
+                    },
+                    "tags": {
+                      "type": "array",
+                      "items": {},
+                      "description": "tags"
+                    },
+                    "flowName": {
+                      "type": "string",
+                      "description": "flowName"
+                    },
+                    "status": {
+                      "type": "string",
+                      "description": "status"
+                    }
+                  },
+                  "description": "response"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api-request-applicant-check": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Request Applicant Check",
+        "description": "Use this action to let us know you've finished uploading documents to an applicant, and to start the verification process. Note: only use this if you are uploading documents. Verifications using a link don't need to use this action.",
```

Comment:
please keep all operations descriptions short to fit in the text box. please see our [docs](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).